modules/ROOT/pages/activity-tracking-edit.adoc

```diff
@@ -0,0 +1,58 @@
+= Activity Tracking
+
+B2B transactions often communicate contractual obligations between companies. Due to the sensitivity of this information, it is desirable (and often necessary) to track transmissions and store both transmission content and metadata.
+
+Anypoint Partner Manager enables you to monitor trading partner transmission activity. Activity is the complete set of end-to-end events related to transmission metadata, messages, and errors that are involved in processing a transmission.
+
+Metadata includes things like the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+
+Common uses for this metadata include:
+
+* Nonrepudiation
+* Auditing
+* Correlation of acknowledgments
+* Error reporting
+
+With activity tracking, you can:
+
+* Search for message metadata (such as partner name, message type, filename, control number, endpoint, or date received).
+* View and download the original payload related to the transmission.
+* Review errors and acknowledgment payloads.
+
+== Track Message Flow Activity
+
+To track message flow activity:
+
+. Open Anypoint Partner Manager.
+. Select *Activity* from the left navigation menu.
+. Locate a transmission you are interested in viewing. 
+. Use the search box and filters to find the transmission based on criteria such as:
++
+* Trading partner name
+* Message type
+* Filename
+* Control number
+* Endpoint
+* Date the file was received
+. Select the transmission that matches your search criteria and review the summary information. +
+The summary information contains:
```

Comment:
This c/b omitted by ending prev. sentence with a colon and going straight to list....